starter-code/public/scripts/controllers/articleController.js

```diff
@@ -5,7 +5,11 @@
   // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
-
+  articleController.index = function () {
+    // Article.fetchAll(articleView.initIndexPage);
+    $('main > section').hide();
```

Comment:
Nice jQuery statements. Though do try to be more specific when targeting elements. Note that in index.html the sections have a class of tab-content